tests/test_cli.py

```diff
@@ -169,3 +169,22 @@ def test_config_passed_down_to_command_children(
             "params": {"time": 5},
         }
     }
+
+
+def test_invalid_stomp_config_for_listner(runner: CliRunner):
+    result = runner.invoke(main, ["controller", "listen"])
+    assert result.exit_code == 1
```

Comment:
Instead of using the exit code, `result.exception` could be checked instead. Then could raise a custom exception and check that it is the expected type.